pulser-core/pulser/backend/noise_model.py

```diff
@@ -44,12 +44,16 @@ class NoiseModel:
             options:
 
             - "dephasing": Random phase (Z) flip (parametrized
-              by `dephasing_rate`).
+              by `dephasing_rate`), commonly characterized experimentally
+              by the T2 time.
             - "relaxation": Noise due to a decay from the Rydberg to
-              the ground state (parametrized by `relaxation_rate`).
+              the ground state (parametrized by `relaxation_rate`), commonly
+              characterized experimentally by the T1 time.
             - "depolarizing": Quantum noise where the state is
-              turned into a mixed state I/2 with rate
-              `depolarizing_rate`.
+              turned into a mixed state I/2 with rate `depolarizing_rate`.
+              While it does not describe a physical phenomenon, it is a
+              commonly used tool to test the system under a uniform
+              combination of phase and amplitude noise.
```

Comment:
You might want to use relaxation instead of amplitude ? We don't use amplitude above :)

Reply:
It's not really relaxation though, it's rather X (bit) and Z (sign) flips but those sounded too "digital"

Reply:
Yes I thought about it afterwards. It's fine by me how it is now !

Reply:
Tbh I don't like it myself, we already have another type of "amplitude" noise so this is bound to be confusing. I'll change it